RAPP/build/bin/generate_deps.sh

```diff
@@ -4,6 +4,7 @@ includes="-I ${rapproot}/src/include"
 rappsrc="${rapproot}/src/*/*.f90 ${rapproot}/src/*/*.F90 ${rapproot}/src/*/*.c"
 /bin/rm -f dependency.mk
 ./sfmakedepend.pl ${includes} -f dependency.mk ${rappsrc}
-sed -i s@hdf5.mod@@g dependency.mk
-sed -i s@netcdf.mod@@g dependency.mk
+sed -i .old s@hdf5.mod@@g dependency.mk
+sed -i .old 's@[^_]netcdf.mod@ @g' dependency.mk
```

Comment:
Yeah, netcdf.mod is at the end of a line in dependency.mk a couple of times, and I was worried the line ending or return would be messed up if it was replaced with nothing. So I replaced it with a space. Not sure if it's strictly necessary though.